recipe/build.sh

```diff
@@ -1,7 +1,17 @@
 #!/bin/bash
+if [ `uname` == "Linux" ]; then
+    LD_SUFFIX="so"
+elif [ `uname` == "Darwin" ]; then
+    LD_SUFFIX="dylib"
+else
+    LD_SUFFIX="DLL"
```

Comment:
Assuming that everything else is Windows can be dangerous for other people than conda-forge that use this recipe to build on various platforms.